bot/exts/fun/uwu.py

```diff
@@ -105,15 +107,25 @@ def _uwuify(self, input_string: str, *, stutter_strength: float = 0.2, emoji_str
         return input_string
 
     @commands.command(name="uwu", aliases=("uwuwize", "uwuify",))
-    async def uwu_command(self, ctx: Context, *, text: clean_content(fix_channel_mentions=True)) -> None:
+    async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> None:
```

Comment:
Shouldn't we make this an optional `clean_content` as it was originally, or call the converter in the body of the function?

Reply:
Seeing what we do below with the replies, we should probably call it in the body itself after getting the final text we'll be working with.

Reply:
fa5c359e7b4adc36d963d951c5ae79908c82afa5